openflow_input/oxm_nicira_ct

```diff
@@ -0,0 +1,175 @@
+// Copyright 2013, Big Switch Networks, Inc.
+//
+// LoxiGen is licensed under the Eclipse Public License, version 1.0 (EPL), with
+// the following special exception:
+//
+// LOXI Exception
+//
+// As a special exception to the terms of the EPL, you may distribute libraries
+// generated by LoxiGen (LoxiGen Libraries) under the terms of your choice, provided
+// that copyright and licensing notices generated by LoxiGen are not altered or removd
+// from the LoxiGen Libraries and the notice provided below is (i) included in
+// the LoxiGen Libraries, if distributed in source code form and (ii) included in any
+// documentation for the LoxiGen Libraries, if distributed in binary form.
+//
+// Notice: "Copyright 2013, Big Switch Networks, Inc. This library was generated by t"
+//
+// You may not use this file except in compliance with the EPL or LOXI Exception. Youn
+// a copy of the EPL at:
+//
+// http://www.eclipse.org/legal/epl-v10.html
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// EPL for the specific language governing permissions and limitations
+// under the EPL.
+
+
+#version 3
+#version 4
+#version 5
+#version 6
+
+
+// Nicira extension for Connection Tracking State (105)
+struct of_oxm_conn_tracking_state : of_oxm {
+	uint32_t type_len == 0x0001d204;
+	uint32_t value;
+};
+
+struct of_oxm_conn_tracking_state_masked : of_oxm {
+	uint32_t type_len == 0x0001d308;
+	uint32_t value;
+	uint32_t value_mask;
+};
+
+
+// Nicira extension for Connection Trakcing Zone (106)
+struct of_oxm_conn_tracking_zone : of_oxm {
+	uint32_t type_len == 0x0001d402;
+	uint16_t value;
+};
+
+struct of_oxm_conn_tracking_zone_masked : of_oxm {
+  uint32_t type_len == 0x0001d504;
+  uint16_t value;
+  uint16_t value_mask;
+};
+
+
+// Nicira extension for Connection Tracking Mark (107)
+struct of_oxm_conn_tracking_mark : of_oxm {
+	uint32_t type_len == 0x0001d604;
+	uint32_t value;
+};
+
+struct of_oxm_conn_tracking_mark_masked : of_oxm {
+	uint32_t type_len == 0x0001d708;
+	uint32_t value;
+	uint32_t value_mask;
+};
+
+
+// Nicira extension for Connection Tracking Label (108)
+struct of_oxm_conn_tracking_label : of_oxm {
+	uint32_t type_len == 0x0001d810;
+	of_bitmap_128_t value;
```

Comment:
@QingWang0909 is a bitmap the best type to use here?

Reply:
@rizard , there is no "uint128_t" type to fit here, at least I didn't find before. If not bitmap, what else type you might suggest? I just found a 'u128', maybe that is a better type?

BTW, I fixed indentation issue in file.

Reply:
@QingWang0909 yes, u128 should be it

Reply:
@rizard , looks like using u128 needs more code modification. I'll of course do that if I need to. I am not exactly sure why u128 is better than bitmap though, do you mind to check out this one? (https://github.com/floodlight/loxigen/blob/master/openflow_input/bsn_in_ports) -- they uses of_bitmap_128_t.

Reply:
@QingWang0909 bsn_in_ports is a bitmap. Your use case is not a bitmap AFAIK. It's best for semantics if you use u128 I think, since the tracking label is just a 128-bit number, not a bitmap.